### PR TITLE
Missing Parameter ManagementPointTypeNetwork

### PR DESCRIPTION
**Pull request by allanhirt**
Added in WS2019.

### CODE REVIEW

**Comment by e0i** [comment]
@allanhirt Thank you for this contribution.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by get-itips** [comment]
On my Windows Server 2019 the parameter name is different and also the accepted values are different.

````
 -ManagementPointNetworkType
Type: AdminAccessPointResType
Parameter Sets: (All)
Aliases: apnt
Accepted values: Automatic, Singleton, Distributed

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
````

Can you check ? Thanks

File: docset/windows/failoverclusters/New-Cluster.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Creates a new failover cluster.
 ```
 New-Cluster [-Name] <String> [-Node <StringCollection>] [-StaticAddress <StringCollection>]
  [-IgnoreNetwork <StringCollection>] [-NoStorage] [-S2D] [-AdministrativeAccessPoint <AdminAccessPoint>]
- [-Force] [<CommonParameters>]
+ [-Force] [-ManagementPointTypeNetwork] [<CommonParameters>]
```
Comment (dariomws) [inline docset/windows/failoverclusters/New-Cluster.md:31]:
```suggestion
 [-Force] [-ManagementPointNetworkType] [<CommonParameters>]
```

File: docset/windows/failoverclusters/New-Cluster.md [hunk 2]
```diff
@@ -168,6 +168,21 @@ Accept pipeline input: False
...
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
```
Comment (dariomws) [inline docset/windows/failoverclusters/New-Cluster.md:184]:
```suggestion
Type: AdminAccessPointResType
Parameter Sets: (All)
Aliases: apnt
Accepted values: Automatic, Singleton, Distributed

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

**Comment by dariomws** [comment]
Agree with @get-itips. Also, not sure about the description: Type of name?

**Comment by allanhirt** [comment]
It’s creates the WSFC name as a distributed network name.

CCing John Marlin.

**Comment by e0i** [comment]
@allanhirt  Windows Server 2019 is out of scope for this documentation repository. I am sorry that I have missed this detail during reviews. Thank you again for your contributions.

**Pull request closed without merging**